lib-common/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'java'
+apply plugin: 'jacoco'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'me.tatarka.retrolambda'
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+
+project.archivesBaseName = 'common'
```

Comment:
I'd prefer name this module "java-common". We can have scala-common in the future.